.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,24 @@
+name: Benchmarks
+on:
+  schedule:
+#   Cron uses UTC - now this time is 00:00, during winter it won't be
+    - cron: '0 22 * * *'
+#  push:
+jobs:
+  benchmarks:
+    name: Benchmarks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "openjdk@1.11"
+      - name: Run benchmarks
+        run: sbt "benchmarks/jmh:run -i 8 -wi 3 -f1 -t2 .SampleSpelBenchmark -rf csv"
```

Comment:
I wonder, maybe it's better to use just one thread? (-t1) 
And let's check how long will it take for all benchmarks to run?

Reply:
At first glance there are minor/no differences between -t1 or -t2. With same parameters full benchmarks should be completed about 40 minutes https://github.com/apanczyk/nussknacker/actions/runs/1127313049.